validate-crt-with-docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+CERT_FILE="`pwd`/generated-certs/crt.pem"
```

Comment:
In the README you say

> if you want to change the certificate file to validate, just change the env variable CERT_FILE inside validate-crt-with-docker.sh and launch the script.

I think this is not so elegant because it requires to modify a file under revision. A solution could be something like that

```
CERT_FILE=${CERT_FILE:="$(pwd)/generated-certs/crt.pem"}
```

This will allow you to update `CERT_FILE` at runtime like in the following

```
CERT_FILE=/abs/path/to/my/file.pem ./validate-crt-with-docker.sh
```

Reply:
done! 👍🏻
@psmiraglia please resolve the conversation if the edits are ok!